__tests__/app.test.js

```diff
@@ -396,3 +393,43 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+// -------------------------------------------------------------------------
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: Deletes a comment with the provided comment_id", () => {
+    return request(app)
+      .delete("/api/comments/3")
+      .expect(204)
+      .then(({ body }) => {
+        expect(body).toEqual({});
+
+        return request(app)
+          .delete("/api/comments/3")
+          .expect(404)
+          .then(({ body: { msg } }) => {
+            expect(msg).toBe("Comment not found");
+          });
+      });
+  });
+
+  test("404: Sends an appropriate status and error message when given a valid but non-existent comment_id", () => {
+    return request(app)
+      .delete("/api/comments/1103")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Comment not found");
+      });
+  });
+
+  test.only("400: Sends an appropriate status and error message when given an invalid comment_id", () => {
```

Comment:
This is the only test running. I can't be sure this actually works when you do this. As it is a short ticket I'll be able to infer for myself, however If this appears again the pr will be rejected.